[2023-05-12T00:00:03.046+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: user_processing.is_api_available scheduled__2023-05-11T00:00:00+00:00 [queued]>
[2023-05-12T00:00:03.052+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: user_processing.is_api_available scheduled__2023-05-11T00:00:00+00:00 [queued]>
[2023-05-12T00:00:03.052+0000] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2023-05-12T00:00:03.052+0000] {taskinstance.py:1289} INFO - Starting attempt 1 of 1
[2023-05-12T00:00:03.053+0000] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2023-05-12T00:00:03.061+0000] {taskinstance.py:1309} INFO - Executing <Task(HttpSensor): is_api_available> on 2023-05-11 00:00:00+00:00
[2023-05-12T00:00:03.065+0000] {standard_task_runner.py:55} INFO - Started process 160494 to run task
[2023-05-12T00:00:03.067+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'is_api_available', 'scheduled__2023-05-11T00:00:00+00:00', '--job-id', '415', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpgu_54poa']
[2023-05-12T00:00:03.068+0000] {standard_task_runner.py:83} INFO - Job 415: Subtask is_api_available
[2023-05-12T00:00:03.114+0000] {task_command.py:389} INFO - Running <TaskInstance: user_processing.is_api_available scheduled__2023-05-11T00:00:00+00:00 [running]> on host 1f05588790e0
[2023-05-12T00:00:03.162+0000] {taskinstance.py:1518} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=is_api_available
AIRFLOW_CTX_EXECUTION_DATE=2023-05-11T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-05-11T00:00:00+00:00
[2023-05-12T00:00:03.165+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T00:00:03.170+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T00:00:04.053+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T00:00:04.053+0000] {http.py:171} ERROR - Not Found
[2023-05-12T00:01:04.113+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T00:01:04.118+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T00:01:05.399+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T00:01:05.400+0000] {http.py:171} ERROR - Not Found
[2023-05-12T00:02:05.435+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T00:02:05.440+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T00:02:05.937+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T00:02:05.937+0000] {http.py:171} ERROR - Not Found
[2023-05-12T00:03:05.993+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T00:03:05.998+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T00:03:06.470+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T00:03:06.470+0000] {http.py:171} ERROR - Not Found
[2023-05-12T00:04:06.486+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T00:04:06.492+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T00:04:06.987+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T00:04:06.987+0000] {http.py:171} ERROR - Not Found
[2023-05-12T00:05:07.048+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T00:05:07.053+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T00:05:08.750+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T00:05:08.751+0000] {http.py:171} ERROR - Not Found
[2023-05-12T00:06:08.808+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T00:06:08.813+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T00:06:09.290+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T00:06:09.290+0000] {http.py:171} ERROR - Not Found
[2023-05-12T00:07:09.342+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T00:07:09.348+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T00:07:10.049+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T00:07:10.049+0000] {http.py:171} ERROR - Not Found
[2023-05-12T00:08:10.110+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T00:08:10.115+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T00:08:10.615+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T00:08:10.615+0000] {http.py:171} ERROR - Not Found
[2023-05-12T00:09:10.676+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T00:09:10.682+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T00:09:11.905+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T00:09:11.905+0000] {http.py:171} ERROR - Not Found
[2023-05-12T00:10:11.965+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T00:10:11.970+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T00:10:12.456+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T00:10:12.456+0000] {http.py:171} ERROR - Not Found
[2023-05-12T00:11:12.517+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T00:11:12.522+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T00:11:12.993+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T00:11:12.994+0000] {http.py:171} ERROR - Not Found
[2023-05-12T00:12:13.053+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T00:12:13.057+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T00:12:13.540+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T00:12:13.540+0000] {http.py:171} ERROR - Not Found
[2023-05-12T00:13:13.601+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T00:13:13.606+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T00:13:14.084+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T00:13:14.084+0000] {http.py:171} ERROR - Not Found
[2023-05-12T00:14:14.145+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T00:14:14.150+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T00:14:14.624+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T00:14:14.624+0000] {http.py:171} ERROR - Not Found
[2023-05-12T00:15:14.685+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T00:15:14.690+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T00:15:15.201+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T00:15:15.201+0000] {http.py:171} ERROR - Not Found
[2023-05-12T00:16:15.261+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T00:16:15.265+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T00:16:16.507+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T00:16:16.507+0000] {http.py:171} ERROR - Not Found
[2023-05-12T00:17:16.568+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T00:17:16.574+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T00:17:17.043+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T00:17:17.044+0000] {http.py:171} ERROR - Not Found
[2023-05-12T00:18:17.103+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T00:18:17.108+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T00:18:17.577+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T00:18:17.577+0000] {http.py:171} ERROR - Not Found
[2023-05-12T00:19:17.635+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T00:19:17.640+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T00:19:18.136+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T00:19:18.137+0000] {http.py:171} ERROR - Not Found
[2023-05-12T00:20:18.197+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T00:20:18.204+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T00:20:19.503+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T00:20:19.503+0000] {http.py:171} ERROR - Not Found
[2023-05-12T00:21:19.561+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T00:21:19.566+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T00:21:20.802+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T00:21:20.803+0000] {http.py:171} ERROR - Not Found
[2023-05-12T00:22:20.864+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T00:22:20.869+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T00:22:21.387+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T00:22:21.387+0000] {http.py:171} ERROR - Not Found
[2023-05-12T00:23:21.447+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T00:23:21.452+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T00:23:21.967+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T00:23:21.967+0000] {http.py:171} ERROR - Not Found
[2023-05-12T00:24:22.029+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T00:24:22.034+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T00:24:22.521+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T00:24:22.521+0000] {http.py:171} ERROR - Not Found
[2023-05-12T00:25:22.581+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T00:25:22.585+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T00:25:23.089+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T00:25:23.089+0000] {http.py:171} ERROR - Not Found
[2023-05-12T00:26:23.149+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T00:26:23.153+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T00:26:24.424+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T00:26:24.424+0000] {http.py:171} ERROR - Not Found
[2023-05-12T00:27:24.484+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T00:27:24.489+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T00:27:24.968+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T00:27:24.968+0000] {http.py:171} ERROR - Not Found
[2023-05-12T00:28:25.029+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T00:28:25.033+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T00:28:25.517+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T00:28:25.517+0000] {http.py:171} ERROR - Not Found
[2023-05-12T00:29:25.578+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T00:29:25.584+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T00:29:26.075+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T00:29:26.076+0000] {http.py:171} ERROR - Not Found
[2023-05-12T00:30:26.137+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T00:30:26.142+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T00:30:27.413+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T00:30:27.413+0000] {http.py:171} ERROR - Not Found
[2023-05-12T00:31:27.465+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T00:31:27.470+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T00:31:27.969+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T00:31:27.970+0000] {http.py:171} ERROR - Not Found
[2023-05-12T00:32:28.022+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T00:32:28.027+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T00:32:28.517+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T00:32:28.518+0000] {http.py:171} ERROR - Not Found
[2023-05-12T00:33:28.577+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T00:33:28.582+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T00:33:29.066+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T00:33:29.066+0000] {http.py:171} ERROR - Not Found
[2023-05-12T00:34:29.125+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T00:34:29.129+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T00:34:29.625+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T00:34:29.626+0000] {http.py:171} ERROR - Not Found
[2023-05-12T00:35:29.685+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T00:35:29.690+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T00:35:30.188+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T00:35:30.189+0000] {http.py:171} ERROR - Not Found
[2023-05-12T00:36:30.249+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T00:36:30.253+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T00:36:31.484+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T00:36:31.484+0000] {http.py:171} ERROR - Not Found
[2023-05-12T00:37:31.545+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T00:37:31.550+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T00:37:32.056+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T00:37:32.056+0000] {http.py:171} ERROR - Not Found
[2023-05-12T00:38:32.114+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T00:38:32.119+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T00:38:33.422+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T00:38:33.423+0000] {http.py:171} ERROR - Not Found
[2023-05-12T00:39:33.481+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T00:39:33.487+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T00:39:33.977+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T00:39:33.978+0000] {http.py:171} ERROR - Not Found
[2023-05-12T00:40:33.993+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T00:40:33.998+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T00:40:35.252+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T00:40:35.253+0000] {http.py:171} ERROR - Not Found
[2023-05-12T00:41:35.305+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T00:41:35.310+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T00:41:35.784+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T00:41:35.784+0000] {http.py:171} ERROR - Not Found
[2023-05-12T00:42:35.844+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T00:42:35.849+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T00:42:36.460+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T00:42:36.460+0000] {http.py:171} ERROR - Not Found
[2023-05-12T00:43:36.521+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T00:43:36.526+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T00:43:37.003+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T00:43:37.003+0000] {http.py:171} ERROR - Not Found
[2023-05-12T00:44:37.065+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T00:44:37.070+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T00:44:37.558+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T00:44:37.558+0000] {http.py:171} ERROR - Not Found
[2023-05-12T00:45:37.617+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T00:45:37.621+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T00:45:38.114+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T00:45:38.114+0000] {http.py:171} ERROR - Not Found
[2023-05-12T00:46:38.159+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T00:46:38.164+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T00:46:38.661+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T00:46:38.661+0000] {http.py:171} ERROR - Not Found
[2023-05-12T00:47:38.708+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T00:47:38.714+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T00:47:39.321+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T00:47:39.321+0000] {http.py:171} ERROR - Not Found
[2023-05-12T00:48:39.379+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T00:48:39.384+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T00:48:39.860+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T00:48:39.860+0000] {http.py:171} ERROR - Not Found
[2023-05-12T00:49:39.921+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T00:49:39.925+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T00:49:40.411+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T00:49:40.411+0000] {http.py:171} ERROR - Not Found
[2023-05-12T00:50:40.472+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T00:50:40.477+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T00:50:40.969+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T00:50:40.970+0000] {http.py:171} ERROR - Not Found
[2023-05-12T00:51:40.987+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T00:51:40.992+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T00:51:42.193+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T00:51:42.193+0000] {http.py:171} ERROR - Not Found
[2023-05-12T00:52:42.253+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T00:52:42.258+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T00:52:42.747+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T00:52:42.748+0000] {http.py:171} ERROR - Not Found
[2023-05-12T00:53:42.784+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T00:53:42.789+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T00:53:43.290+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T00:53:43.290+0000] {http.py:171} ERROR - Not Found
[2023-05-12T00:54:43.349+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T00:54:43.353+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T00:54:43.838+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T00:54:43.838+0000] {http.py:171} ERROR - Not Found
[2023-05-12T00:55:43.899+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T00:55:43.904+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T00:55:45.196+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T00:55:45.196+0000] {http.py:171} ERROR - Not Found
[2023-05-12T00:56:45.257+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T00:56:45.261+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T00:56:45.735+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T00:56:45.735+0000] {http.py:171} ERROR - Not Found
[2023-05-12T00:57:45.784+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T00:57:45.789+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T00:57:46.274+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T00:57:46.274+0000] {http.py:171} ERROR - Not Found
[2023-05-12T00:58:46.336+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T00:58:46.341+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T00:58:46.844+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T00:58:46.844+0000] {http.py:171} ERROR - Not Found
[2023-05-12T00:59:46.901+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T00:59:46.906+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T00:59:47.407+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T00:59:47.407+0000] {http.py:171} ERROR - Not Found
[2023-05-12T01:00:47.469+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T01:00:47.474+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T01:00:48.004+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T01:00:48.004+0000] {http.py:171} ERROR - Not Found
[2023-05-12T01:01:48.060+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T01:01:48.065+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T01:01:49.343+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T01:01:49.343+0000] {http.py:171} ERROR - Not Found
[2023-05-12T01:02:49.405+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T01:02:49.410+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T01:02:49.910+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T01:02:49.911+0000] {http.py:171} ERROR - Not Found
[2023-05-12T01:03:49.968+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T01:03:49.973+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T01:03:50.458+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T01:03:50.459+0000] {http.py:171} ERROR - Not Found
[2023-05-12T01:04:50.520+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T01:04:50.525+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T01:04:51.017+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T01:04:51.017+0000] {http.py:171} ERROR - Not Found
[2023-05-12T01:05:51.078+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T01:05:51.084+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T01:05:51.598+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T01:05:51.599+0000] {http.py:171} ERROR - Not Found
[2023-05-12T01:06:51.656+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T01:06:51.661+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T01:06:52.142+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T01:06:52.142+0000] {http.py:171} ERROR - Not Found
[2023-05-12T01:07:52.173+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T01:07:52.179+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T01:07:52.667+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T01:07:52.667+0000] {http.py:171} ERROR - Not Found
[2023-05-12T01:08:52.729+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T01:08:52.735+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T01:08:53.209+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T01:08:53.210+0000] {http.py:171} ERROR - Not Found
[2023-05-12T01:09:53.271+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T01:09:53.277+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T01:09:53.764+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T01:09:53.764+0000] {http.py:171} ERROR - Not Found
[2023-05-12T01:10:53.825+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T01:10:53.829+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T01:10:54.334+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T01:10:54.335+0000] {http.py:171} ERROR - Not Found
[2023-05-12T01:11:54.396+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T01:11:54.401+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T01:11:54.879+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T01:11:54.879+0000] {http.py:171} ERROR - Not Found
[2023-05-12T01:12:54.937+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T01:12:54.942+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T01:12:55.428+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T01:12:55.428+0000] {http.py:171} ERROR - Not Found
[2023-05-12T01:13:55.489+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T01:13:55.493+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T01:13:55.971+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T01:13:55.971+0000] {http.py:171} ERROR - Not Found
[2023-05-12T01:14:56.033+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T01:14:56.037+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T01:14:56.523+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T01:14:56.523+0000] {http.py:171} ERROR - Not Found
[2023-05-12T01:15:56.582+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T01:15:56.588+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T01:15:57.069+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T01:15:57.070+0000] {http.py:171} ERROR - Not Found
[2023-05-12T01:16:57.124+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T01:16:57.130+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T01:16:57.611+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T01:16:57.611+0000] {http.py:171} ERROR - Not Found
[2023-05-12T01:17:57.671+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T01:17:57.676+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T01:17:58.177+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T01:17:58.177+0000] {http.py:171} ERROR - Not Found
[2023-05-12T01:18:58.237+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T01:18:58.241+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T01:18:58.721+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T01:18:58.721+0000] {http.py:171} ERROR - Not Found
[2023-05-12T01:19:58.780+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T01:19:58.785+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T01:19:59.274+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T01:19:59.275+0000] {http.py:171} ERROR - Not Found
[2023-05-12T01:20:59.333+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T01:20:59.337+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T01:20:59.827+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T01:20:59.827+0000] {http.py:171} ERROR - Not Found
[2023-05-12T01:21:59.885+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T01:21:59.889+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T01:22:00.455+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T01:22:00.455+0000] {http.py:171} ERROR - Not Found
[2023-05-12T01:23:00.508+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T01:23:00.513+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T01:23:01.032+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T01:23:01.032+0000] {http.py:171} ERROR - Not Found
[2023-05-12T01:24:01.093+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T01:24:01.098+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T01:24:01.586+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T01:24:01.587+0000] {http.py:171} ERROR - Not Found
[2023-05-12T01:25:01.643+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T01:25:01.648+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T01:25:02.132+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T01:25:02.133+0000] {http.py:171} ERROR - Not Found
[2023-05-12T01:26:02.187+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T01:26:02.192+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T01:26:02.722+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T01:26:02.723+0000] {http.py:171} ERROR - Not Found
[2023-05-12T01:27:02.781+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T01:27:02.784+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T01:27:03.284+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T01:27:03.284+0000] {http.py:171} ERROR - Not Found
[2023-05-12T01:28:03.344+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T01:28:03.349+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T01:28:04.638+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T01:28:04.638+0000] {http.py:171} ERROR - Not Found
[2023-05-12T01:29:04.693+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T01:29:04.697+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T01:29:05.186+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T01:29:05.186+0000] {http.py:171} ERROR - Not Found
[2023-05-12T01:30:05.244+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T01:30:05.250+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T01:30:05.760+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T01:30:05.761+0000] {http.py:171} ERROR - Not Found
[2023-05-12T01:31:05.821+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T01:31:05.825+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T01:31:06.325+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T01:31:06.325+0000] {http.py:171} ERROR - Not Found
[2023-05-12T01:32:06.385+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T01:32:06.390+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T01:32:06.900+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T01:32:06.901+0000] {http.py:171} ERROR - Not Found
[2023-05-12T01:33:06.960+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T01:33:06.965+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T01:33:07.456+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T01:33:07.457+0000] {http.py:171} ERROR - Not Found
[2023-05-12T01:34:07.517+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T01:34:07.522+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T01:34:08.001+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T01:34:08.001+0000] {http.py:171} ERROR - Not Found
[2023-05-12T01:35:08.062+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T01:35:08.067+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T01:35:08.549+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T01:35:08.550+0000] {http.py:171} ERROR - Not Found
[2023-05-12T01:36:08.610+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T01:36:08.616+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T01:36:09.130+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T01:36:09.130+0000] {http.py:171} ERROR - Not Found
[2023-05-12T01:37:09.158+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T01:37:09.163+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T01:37:09.676+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T01:37:09.676+0000] {http.py:171} ERROR - Not Found
[2023-05-12T01:38:09.737+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T01:38:09.742+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T01:38:10.242+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T01:38:10.242+0000] {http.py:171} ERROR - Not Found
[2023-05-12T01:39:10.302+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T01:39:10.307+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T01:39:10.792+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T01:39:10.792+0000] {http.py:171} ERROR - Not Found
[2023-05-12T01:40:10.850+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T01:40:10.855+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T01:40:11.314+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T01:40:11.314+0000] {http.py:171} ERROR - Not Found
[2023-05-12T01:41:11.372+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T01:41:11.378+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T01:41:11.870+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T01:41:11.870+0000] {http.py:171} ERROR - Not Found
[2023-05-12T01:42:11.928+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T01:42:11.933+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T01:42:12.418+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T01:42:12.418+0000] {http.py:171} ERROR - Not Found
[2023-05-12T01:43:12.477+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T01:43:12.482+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T01:43:12.982+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T01:43:12.982+0000] {http.py:171} ERROR - Not Found
[2023-05-12T01:44:13.041+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T01:44:13.046+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T01:44:13.534+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T01:44:13.534+0000] {http.py:171} ERROR - Not Found
[2023-05-12T01:45:13.595+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T01:45:13.600+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T01:45:14.130+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T01:45:14.130+0000] {http.py:171} ERROR - Not Found
[2023-05-12T01:46:14.189+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T01:46:14.194+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T01:46:14.682+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T01:46:14.682+0000] {http.py:171} ERROR - Not Found
[2023-05-12T01:47:14.735+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T01:47:14.740+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T01:47:15.226+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T01:47:15.227+0000] {http.py:171} ERROR - Not Found
[2023-05-12T01:48:15.285+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T01:48:15.289+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T01:48:15.791+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T01:48:15.791+0000] {http.py:171} ERROR - Not Found
[2023-05-12T01:49:15.852+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T01:49:15.858+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T01:49:16.351+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T01:49:16.351+0000] {http.py:171} ERROR - Not Found
[2023-05-12T01:50:16.412+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T01:50:16.419+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T01:50:16.910+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T01:50:16.911+0000] {http.py:171} ERROR - Not Found
[2023-05-12T01:51:16.962+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T01:51:16.967+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T01:51:17.458+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T01:51:17.458+0000] {http.py:171} ERROR - Not Found
[2023-05-12T01:52:17.517+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T01:52:17.521+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T01:52:18.024+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T01:52:18.025+0000] {http.py:171} ERROR - Not Found
[2023-05-12T01:53:18.084+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T01:53:18.089+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T01:53:18.592+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T01:53:18.592+0000] {http.py:171} ERROR - Not Found
[2023-05-12T01:54:18.601+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T01:54:18.606+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T01:54:19.084+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T01:54:19.085+0000] {http.py:171} ERROR - Not Found
[2023-05-12T01:55:19.145+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T01:55:19.149+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T01:55:19.643+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T01:55:19.643+0000] {http.py:171} ERROR - Not Found
[2023-05-12T01:56:19.697+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T01:56:19.702+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T01:56:20.327+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T01:56:20.327+0000] {http.py:171} ERROR - Not Found
[2023-05-12T01:57:20.389+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T01:57:20.394+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T01:57:20.879+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T01:57:20.879+0000] {http.py:171} ERROR - Not Found
[2023-05-12T01:58:20.937+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T01:58:20.941+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T01:58:21.407+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T01:58:21.407+0000] {http.py:171} ERROR - Not Found
[2023-05-12T01:59:21.468+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T01:59:21.473+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T01:59:21.980+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T01:59:21.980+0000] {http.py:171} ERROR - Not Found
[2023-05-12T02:00:22.041+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T02:00:22.046+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T02:00:22.549+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T02:00:22.549+0000] {http.py:171} ERROR - Not Found
[2023-05-12T02:01:22.608+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T02:01:22.614+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T02:01:23.102+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T02:01:23.103+0000] {http.py:171} ERROR - Not Found
[2023-05-12T02:02:23.164+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T02:02:23.169+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T02:02:23.680+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T02:02:23.680+0000] {http.py:171} ERROR - Not Found
[2023-05-12T02:03:23.740+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T02:03:23.746+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T02:03:24.227+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T02:03:24.227+0000] {http.py:171} ERROR - Not Found
[2023-05-12T02:04:24.284+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T02:04:24.289+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T02:04:24.790+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T02:04:24.791+0000] {http.py:171} ERROR - Not Found
[2023-05-12T02:05:24.849+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T02:05:24.854+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T02:05:25.366+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T02:05:25.366+0000] {http.py:171} ERROR - Not Found
[2023-05-12T02:06:25.427+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T02:06:25.432+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T02:06:25.934+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T02:06:25.935+0000] {http.py:171} ERROR - Not Found
[2023-05-12T02:07:25.947+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T02:07:25.953+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T02:07:26.452+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T02:07:26.452+0000] {http.py:171} ERROR - Not Found
[2023-05-12T02:08:26.513+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T02:08:26.518+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T02:08:26.998+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T02:08:26.998+0000] {http.py:171} ERROR - Not Found
[2023-05-12T02:09:27.053+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T02:09:27.057+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T02:09:27.558+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T02:09:27.558+0000] {http.py:171} ERROR - Not Found
[2023-05-12T02:10:27.617+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T02:10:27.622+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T02:10:28.126+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T02:10:28.127+0000] {http.py:171} ERROR - Not Found
[2023-05-12T02:11:28.185+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T02:11:28.190+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T02:11:28.686+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T02:11:28.687+0000] {http.py:171} ERROR - Not Found
[2023-05-12T02:12:28.737+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T02:12:28.742+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T02:12:29.217+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T02:12:29.217+0000] {http.py:171} ERROR - Not Found
[2023-05-12T02:13:29.251+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T02:13:29.255+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T02:13:30.536+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T02:13:30.536+0000] {http.py:171} ERROR - Not Found
[2023-05-12T02:14:30.597+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T02:14:30.601+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T02:14:31.818+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T02:14:31.818+0000] {http.py:171} ERROR - Not Found
[2023-05-12T02:15:31.877+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T02:15:31.882+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T02:15:32.368+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T02:15:32.368+0000] {http.py:171} ERROR - Not Found
[2023-05-12T02:16:32.427+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T02:16:32.433+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T02:16:32.951+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T02:16:32.951+0000] {http.py:171} ERROR - Not Found
[2023-05-12T02:17:33.013+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T02:17:33.018+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T02:17:33.504+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T02:17:33.504+0000] {http.py:171} ERROR - Not Found
[2023-05-12T02:18:33.557+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T02:18:33.562+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T02:18:34.061+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T02:18:34.062+0000] {http.py:171} ERROR - Not Found
[2023-05-12T02:19:34.120+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T02:19:34.125+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T02:19:34.677+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T02:19:34.677+0000] {http.py:171} ERROR - Not Found
[2023-05-12T02:20:34.737+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T02:20:34.742+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T02:20:35.506+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T02:20:35.506+0000] {http.py:171} ERROR - Not Found
[2023-05-12T02:21:35.549+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T02:21:35.553+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T02:21:36.094+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T02:21:36.094+0000] {http.py:171} ERROR - Not Found
[2023-05-12T02:22:36.155+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T02:22:36.161+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T02:22:36.664+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T02:22:36.664+0000] {http.py:171} ERROR - Not Found
[2023-05-12T02:23:36.725+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T02:23:36.729+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T02:23:38.047+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T02:23:38.047+0000] {http.py:171} ERROR - Not Found
[2023-05-12T02:24:38.108+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T02:24:38.113+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T02:24:39.337+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T02:24:39.338+0000] {http.py:171} ERROR - Not Found
[2023-05-12T02:25:39.383+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T02:25:39.388+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T02:25:39.858+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T02:25:39.859+0000] {http.py:171} ERROR - Not Found
[2023-05-12T02:26:39.916+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T02:26:39.921+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T02:26:40.429+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T02:26:40.429+0000] {http.py:171} ERROR - Not Found
[2023-05-12T02:27:40.468+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T02:27:40.474+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T02:27:40.957+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T02:27:40.957+0000] {http.py:171} ERROR - Not Found
[2023-05-12T02:28:41.016+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T02:28:41.021+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T02:28:41.522+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T02:28:41.522+0000] {http.py:171} ERROR - Not Found
[2023-05-12T02:29:41.580+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T02:29:41.585+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T02:29:42.059+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T02:29:42.059+0000] {http.py:171} ERROR - Not Found
[2023-05-12T02:30:42.121+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T02:30:42.125+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T02:30:42.636+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T02:30:42.636+0000] {http.py:171} ERROR - Not Found
[2023-05-12T02:31:42.689+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T02:31:42.695+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T02:31:43.208+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T02:31:43.208+0000] {http.py:171} ERROR - Not Found
[2023-05-12T02:32:43.269+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T02:32:43.273+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T02:32:43.759+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T02:32:43.759+0000] {http.py:171} ERROR - Not Found
[2023-05-12T02:33:43.819+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T02:33:43.824+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T02:33:44.344+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T02:33:44.344+0000] {http.py:171} ERROR - Not Found
[2023-05-12T02:34:44.404+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T02:34:44.409+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T02:34:44.913+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T02:34:44.913+0000] {http.py:171} ERROR - Not Found
[2023-05-12T02:35:44.973+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T02:35:44.978+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T02:35:45.488+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T02:35:45.488+0000] {http.py:171} ERROR - Not Found
[2023-05-12T02:36:45.549+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T02:36:45.554+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T02:36:46.786+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T02:36:46.787+0000] {http.py:171} ERROR - Not Found
[2023-05-12T02:37:46.848+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T02:37:46.855+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T02:37:47.344+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T02:37:47.344+0000] {http.py:171} ERROR - Not Found
[2023-05-12T02:38:47.405+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T02:38:47.410+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T02:38:47.942+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T02:38:47.942+0000] {http.py:171} ERROR - Not Found
[2023-05-12T02:39:48.004+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T02:39:48.009+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T02:39:48.507+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T02:39:48.507+0000] {http.py:171} ERROR - Not Found
[2023-05-12T02:40:48.569+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T02:40:48.574+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T02:40:49.078+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T02:40:49.078+0000] {http.py:171} ERROR - Not Found
[2023-05-12T02:41:49.137+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T02:41:49.142+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T02:41:49.641+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T02:41:49.641+0000] {http.py:171} ERROR - Not Found
[2023-05-12T02:42:49.700+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T02:42:49.705+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T02:42:50.211+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T02:42:50.211+0000] {http.py:171} ERROR - Not Found
[2023-05-12T02:43:50.273+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T02:43:50.277+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T02:43:50.760+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T02:43:50.760+0000] {http.py:171} ERROR - Not Found
[2023-05-12T02:44:50.821+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T02:44:50.826+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T02:44:52.158+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T02:44:52.158+0000] {http.py:171} ERROR - Not Found
[2023-05-12T02:45:52.220+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T02:45:52.226+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T02:45:52.717+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T02:45:52.718+0000] {http.py:171} ERROR - Not Found
[2023-05-12T02:46:52.777+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T02:46:52.781+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T02:46:53.267+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T02:46:53.268+0000] {http.py:171} ERROR - Not Found
[2023-05-12T02:47:53.326+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T02:47:53.331+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T02:47:53.851+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T02:47:53.851+0000] {http.py:171} ERROR - Not Found
[2023-05-12T02:48:53.902+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T02:48:53.907+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T02:48:54.377+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T02:48:54.377+0000] {http.py:171} ERROR - Not Found
[2023-05-12T02:49:54.414+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T02:49:54.420+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T02:49:54.914+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T02:49:54.915+0000] {http.py:171} ERROR - Not Found
[2023-05-12T02:50:54.976+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T02:50:54.981+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T02:50:55.485+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T02:50:55.485+0000] {http.py:171} ERROR - Not Found
[2023-05-12T02:51:55.545+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T02:51:55.549+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T02:51:56.033+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T02:51:56.033+0000] {http.py:171} ERROR - Not Found
[2023-05-12T02:52:56.092+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T02:52:56.099+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T02:52:56.575+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T02:52:56.575+0000] {http.py:171} ERROR - Not Found
[2023-05-12T02:53:56.628+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T02:53:56.634+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T02:53:57.108+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T02:53:57.109+0000] {http.py:171} ERROR - Not Found
[2023-05-12T02:54:57.168+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T02:54:57.173+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T02:54:57.660+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T02:54:57.660+0000] {http.py:171} ERROR - Not Found
[2023-05-12T02:55:57.720+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T02:55:57.726+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T02:55:58.219+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T02:55:58.219+0000] {http.py:171} ERROR - Not Found
[2023-05-12T02:56:58.281+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T02:56:58.287+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T02:56:58.781+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T02:56:58.781+0000] {http.py:171} ERROR - Not Found
[2023-05-12T02:57:58.840+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T02:57:58.846+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T02:58:00.084+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T02:58:00.085+0000] {http.py:171} ERROR - Not Found
[2023-05-12T02:59:00.145+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T02:59:00.150+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T02:59:00.642+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T02:59:00.642+0000] {http.py:171} ERROR - Not Found
[2023-05-12T03:00:00.703+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T03:00:00.708+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T03:00:01.194+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T03:00:01.194+0000] {http.py:171} ERROR - Not Found
[2023-05-12T03:01:01.256+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T03:01:01.261+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T03:01:01.799+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T03:01:01.799+0000] {http.py:171} ERROR - Not Found
[2023-05-12T03:02:01.860+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T03:02:01.865+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T03:02:02.414+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T03:02:02.414+0000] {http.py:171} ERROR - Not Found
[2023-05-12T03:03:02.472+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T03:03:02.477+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T03:03:03.016+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T03:03:03.016+0000] {http.py:171} ERROR - Not Found
[2023-05-12T03:04:03.076+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T03:04:03.081+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T03:04:03.602+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T03:04:03.602+0000] {http.py:171} ERROR - Not Found
[2023-05-12T03:05:03.662+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T03:05:03.666+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T03:05:04.140+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T03:05:04.140+0000] {http.py:171} ERROR - Not Found
[2023-05-12T03:06:04.201+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T03:06:04.206+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T03:06:05.462+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T03:06:05.462+0000] {http.py:171} ERROR - Not Found
[2023-05-12T03:07:05.516+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T03:07:05.521+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T03:07:06.108+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T03:07:06.108+0000] {http.py:171} ERROR - Not Found
[2023-05-12T03:08:06.169+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T03:08:06.174+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T03:08:06.646+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T03:08:06.646+0000] {http.py:171} ERROR - Not Found
[2023-05-12T03:09:06.705+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T03:09:06.711+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T03:09:07.201+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T03:09:07.202+0000] {http.py:171} ERROR - Not Found
[2023-05-12T03:10:07.257+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T03:10:07.261+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T03:10:07.768+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T03:10:07.768+0000] {http.py:171} ERROR - Not Found
[2023-05-12T03:11:07.792+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T03:11:07.797+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T03:11:08.298+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T03:11:08.298+0000] {http.py:171} ERROR - Not Found
[2023-05-12T03:12:08.357+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T03:12:08.362+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T03:12:08.881+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T03:12:08.882+0000] {http.py:171} ERROR - Not Found
[2023-05-12T03:13:08.940+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T03:13:08.946+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T03:13:09.435+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T03:13:09.435+0000] {http.py:171} ERROR - Not Found
[2023-05-12T03:14:09.495+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T03:14:09.500+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T03:14:09.992+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T03:14:09.993+0000] {http.py:171} ERROR - Not Found
[2023-05-12T03:15:10.054+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T03:15:10.059+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T03:15:10.561+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T03:15:10.562+0000] {http.py:171} ERROR - Not Found
[2023-05-12T03:16:10.621+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T03:16:10.626+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T03:16:11.134+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T03:16:11.135+0000] {http.py:171} ERROR - Not Found
[2023-05-12T03:17:11.164+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T03:17:11.169+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T03:17:11.683+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T03:17:11.683+0000] {http.py:171} ERROR - Not Found
[2023-05-12T03:18:11.734+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T03:18:11.739+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T03:18:12.247+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T03:18:12.248+0000] {http.py:171} ERROR - Not Found
[2023-05-12T03:19:12.308+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T03:19:12.313+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T03:19:12.808+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T03:19:12.808+0000] {http.py:171} ERROR - Not Found
[2023-05-12T03:20:12.868+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T03:20:12.873+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T03:20:13.349+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T03:20:13.349+0000] {http.py:171} ERROR - Not Found
[2023-05-12T03:21:13.409+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T03:21:13.414+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T03:21:13.968+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T03:21:13.968+0000] {http.py:171} ERROR - Not Found
[2023-05-12T03:22:14.013+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T03:22:14.017+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T03:22:14.547+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-05-12T03:22:14.547+0000] {http.py:171} ERROR - Not Found
[2023-05-12T03:23:14.600+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-05-12T03:23:14.605+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-05-12T03:23:15.802+0000] {http.py:170} ERROR - HTTP error: 
[2023-05-12T03:23:15.802+0000] {http.py:171} ERROR - <!DOCTYPE html>
<!--[if lt IE 7]> <html class="no-js ie6 oldie" lang="en-US"> <![endif]-->
<!--[if IE 7]>    <html class="no-js ie7 oldie" lang="en-US"> <![endif]-->
<!--[if IE 8]>    <html class="no-js ie8 oldie" lang="en-US"> <![endif]-->
<!--[if gt IE 8]><!--> <html class="no-js" lang="en-US"> <!--<![endif]-->
<head>


<title>randomuser.me | 520: Web server is returning an unknown error</title>
<meta charset="UTF-8" />
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
<meta http-equiv="X-UA-Compatible" content="IE=Edge" />
<meta name="robots" content="noindex, nofollow" />
<meta name="viewport" content="width=device-width,initial-scale=1" />
<link rel="stylesheet" id="cf_styles-css" href="/cdn-cgi/styles/main.css" />


</head>
<body>
<div id="cf-wrapper">

    

    <div id="cf-error-details" class="p-0">
        <header class="mx-auto pt-10 lg:pt-6 lg:px-8 w-240 lg:w-full mb-8">
            <h1 class="inline-block sm:block sm:mb-2 font-light text-60 lg:text-4xl text-black-dark leading-tight mr-2">
              
              <span class="inline-block">Web server is returning an unknown error</span>
              <span class="code-label">Error code 520</span>
            </h1>
            <div>
               Visit <a href="https://www.cloudflare.com/5xx-error-landing?utm_source=errorcode_520&utm_campaign=randomuser.me" target="_blank" rel="noopener noreferrer">cloudflare.com</a> for more information.
            </div>
            <div class="mt-3">2023-05-12 03:23:15 UTC</div>
        </header>
        
        <div class="my-8 bg-gradient-gray">
            <div class="w-240 lg:w-full mx-auto">
                <div class="clearfix md:px-8">
                  
<div id="cf-browser-status" class=" relative w-1/3 md:w-full py-15 md:p-0 md:py-8 md:text-left md:border-solid md:border-0 md:border-b md:border-gray-400 overflow-hidden float-left md:float-none text-center">
  <div class="relative mb-10 md:m-0">
    
    <span class="cf-icon-browser block md:hidden h-20 bg-center bg-no-repeat"></span>
    <span class="cf-icon-ok w-12 h-12 absolute left-1/2 md:left-auto md:right-0 md:top-0 -ml-6 -bottom-4"></span>
    
  </div>
  <span class="md:block w-full truncate">You</span>
  <h3 class="md:inline-block mt-3 md:mt-0 text-2xl text-gray-600 font-light leading-1.3">
    
    Browser
    
  </h3>
  <span class="leading-1.3 text-2xl text-green-success">Working</span>
</div>

<div id="cf-cloudflare-status" class=" relative w-1/3 md:w-full py-15 md:p-0 md:py-8 md:text-left md:border-solid md:border-0 md:border-b md:border-gray-400 overflow-hidden float-left md:float-none text-center">
  <div class="relative mb-10 md:m-0">
    <a href="https://www.cloudflare.com/5xx-error-landing?utm_source=errorcode_520&utm_campaign=randomuser.me" target="_blank" rel="noopener noreferrer">
    <span class="cf-icon-cloud block md:hidden h-20 bg-center bg-no-repeat"></span>
    <span class="cf-icon-ok w-12 h-12 absolute left-1/2 md:left-auto md:right-0 md:top-0 -ml-6 -bottom-4"></span>
    </a>
  </div>
  <span class="md:block w-full truncate">Singapore</span>
  <h3 class="md:inline-block mt-3 md:mt-0 text-2xl text-gray-600 font-light leading-1.3">
    <a href="https://www.cloudflare.com/5xx-error-landing?utm_source=errorcode_520&utm_campaign=randomuser.me" target="_blank" rel="noopener noreferrer">
    Cloudflare
    </a>
  </h3>
  <span class="leading-1.3 text-2xl text-green-success">Working</span>
</div>

<div id="cf-host-status" class="cf-error-source relative w-1/3 md:w-full py-15 md:p-0 md:py-8 md:text-left md:border-solid md:border-0 md:border-b md:border-gray-400 overflow-hidden float-left md:float-none text-center">
  <div class="relative mb-10 md:m-0">
    
    <span class="cf-icon-server block md:hidden h-20 bg-center bg-no-repeat"></span>
    <span class="cf-icon-error w-12 h-12 absolute left-1/2 md:left-auto md:right-0 md:top-0 -ml-6 -bottom-4"></span>
    
  </div>
  <span class="md:block w-full truncate">randomuser.me</span>
  <h3 class="md:inline-block mt-3 md:mt-0 text-2xl text-gray-600 font-light leading-1.3">
    
    Host
    
  </h3>
  <span class="leading-1.3 text-2xl text-red-error">Error</span>
</div>

                </div>
              
            </div>
        </div>

        <div class="w-240 lg:w-full mx-auto mb-8 lg:px-8">
            <div class="clearfix">
                <div class="w-1/2 md:w-full float-left pr-6 md:pb-10 md:pr-0 leading-relaxed">
                    <h2 class="text-3xl font-normal leading-1.3 mb-4">What happened?</h2>
                    <p>There is an unknown connection issue between Cloudflare and the origin web server. As a result, the web page can not be displayed.</p>
                </div>
              
                <div class="w-1/2 md:w-full float-left leading-relaxed">
                    <h2 class="text-3xl font-normal leading-1.3 mb-4">What can I do?</h2>
                          <h3 class="text-15 font-semibold mb-2">If you are a visitor of this website:</h3>
      <p class="mb-6">Please try again in a few minutes.</p>

      <h3 class="text-15 font-semibold mb-2">If you are the owner of this website:</h3>
      <p><span>There is an issue between Cloudflare's cache and your origin web server. Cloudflare monitors for these errors and automatically investigates the cause. To help support the investigation, you can pull the corresponding error log from your web server and submit it our support team.  Please include the Ray ID (which is at the bottom of this error page).</span> <a rel="noopener noreferrer" href="https://support.cloudflare.com/hc/en-us/articles/200171936-Error-520">Additional troubleshooting resources</a>.</p>
                </div>
            </div>
              
        </div>

        <div class="cf-error-footer cf-wrapper w-240 lg:w-full py-10 sm:py-4 sm:px-8 mx-auto text-center sm:text-left border-solid border-0 border-t border-gray-300">
  <p class="text-13">
    <span class="cf-footer-item sm:block sm:mb-1">Cloudflare Ray ID: <strong class="font-semibold">7c5f8ab97bc24733</strong></span>
    <span class="cf-footer-separator sm:hidden">&bull;</span>
    <span id="cf-footer-item-ip" class="cf-footer-item hidden sm:block sm:mb-1">
      Your IP:
      <button type="button" id="cf-footer-ip-reveal" class="cf-footer-ip-reveal-btn">Click to reveal</button>
      <span class="hidden" id="cf-footer-ip">121.244.55.198</span>
      <span class="cf-footer-separator sm:hidden">&bull;</span>
    </span>
    <span class="cf-footer-item sm:block sm:mb-1"><span>Performance &amp; security by</span> <a rel="noopener noreferrer" href="https://www.cloudflare.com/5xx-error-landing?utm_source=errorcode_520&utm_campaign=randomuser.me" id="brand_link" target="_blank">Cloudflare</a></span>
    
  </p>
  <script>(function(){function d(){var b=a.getElementById("cf-footer-item-ip"),c=a.getElementById("cf-footer-ip-reveal");b&&"classList"in b&&(b.classList.remove("hidden"),c.addEventListener("click",function(){c.classList.add("hidden");a.getElementById("cf-footer-ip").classList.remove("hidden")}))}var a=document;document.addEventListener&&a.addEventListener("DOMContentLoaded",d)})();</script>
</div><!-- /.error-footer -->


    </div>
</div>
</body>
</html>


[2023-05-12T03:23:15.809+0000] {taskinstance.py:1776} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/http/hooks/http.py", line 168, in check_response
    response.raise_for_status()
  File "/home/airflow/.local/lib/python3.7/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 520 Server Error:  for url: https://randomuser.me/https://randomuser.me/

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/sensors/base.py", line 199, in execute
    poke_return = self.poke(context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/http/sensors/http.py", line 137, in poke
    raise exc
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/http/sensors/http.py", line 128, in poke
    extra_options=self.extra_options,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/http/hooks/http.py", line 158, in run
    return self.run_and_check(session, prepped_request, extra_options)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/http/hooks/http.py", line 211, in run_and_check
    self.check_response(response)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/http/hooks/http.py", line 172, in check_response
    raise AirflowException(str(response.status_code) + ":" + response.reason)
airflow.exceptions.AirflowException: 520:
[2023-05-12T03:23:15.812+0000] {taskinstance.py:1332} INFO - Marking task as FAILED. dag_id=user_processing, task_id=is_api_available, execution_date=20230511T000000, start_date=20230512T000003, end_date=20230512T032315
[2023-05-12T03:23:15.821+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 415 for task is_api_available (520:; 160494)
[2023-05-12T03:23:16.037+0000] {local_task_job.py:212} INFO - Task exited with return code 1
[2023-05-12T03:23:16.056+0000] {taskinstance.py:2596} INFO - 0 downstream tasks scheduled from follow-on schedule check
